### PR TITLE
merge markGraphnodesDirty and markGraphNodeDirty

### PR DESCRIPTION
**Pull request by zora-wang**
@junmendoza  
please help to review it thank you very much ~~~

MarkGraphNodeDirty method is deleted.
Only MarGraphNodesDirty is used

The relevant task is here http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6966

### CODE REVIEW

**Comment by junmendoza** [comment]
@zora-wang looks good, i will merge it after we confirm there are no regressions in the entire test - Dynamo+language

File: src/Engine/ProtoCore/AssociativeGraph.cs [hunk 1]
```diff
@@ -524,7 +497,7 @@ public static AssociativeGraph.GraphNode MarkGraphNodeDirty(RuntimeCore runtimeC
         /// <param name="core"></param>
         /// <param name="nodeList"></param>
         /// <returns></returns>
-        public static AssociativeGraph.GraphNode MarkGraphNodesDirty(Core core, IEnumerable<AST.AssociativeAST.AssociativeNode> nodeList)
+        public static AssociativeGraph.GraphNode MarkGraphNodesDirty(RuntimeCore core, IEnumerable<AST.AssociativeAST.AssociativeNode> nodeList)
```
Comment (junmendoza) [inline src/Engine/ProtoCore/AssociativeGraph.cs]: Can you rename the function from 'MarkGraphNodesDirty' to 'MarkGraphNodesDirtyAtGlobalScope'

**Comment by zora-wang** [comment]
@junmendoza  The name has been updated to MarkGraphNodesDirtyAtGlobalScope

**Comment by junmendoza** [comment]
LGTM

**Pull request merged**